src/libtls/tls_crypto.h

```diff
@@ -669,4 +669,12 @@ tls_crypto_t *tls_crypto_create(tls_t *tls, tls_cache_t *cache);
 int tls_crypto_get_supported_suites(bool null, tls_version_t version,
 									tls_cipher_suite_t **suites);
 
+/**
+ * Get a list of all supported TLS ECDH groups.
```

Comment:
```suggestion
 * Get a list of all supported TLS DH groups.
```
See the other comment regarding curves vs. groups, would also affect other comments here and the naming of the function/argument.